app/components/form_components/schools_autocomplete/script.js

```diff
@@ -0,0 +1,72 @@
+import accessibleAutocomplete from 'accessible-autocomplete'
+import { nodeListForEach } from 'govuk-frontend/govuk/common'
+
+const $allAutocompleteElements = document.querySelectorAll('[data-module="app-schools-autocomplete"]')
+const idElement = document.getElementById('lead-school-id')
+
+let statusMessage = ' '
+
+const guard = (data) => {
+  if (data === undefined) {
+    return new Error('An error occured')
+  }
+
+  return data
+}
+
+const mapToSchools = (data) => data.schools
+
+const tryUpdateStatusMessage = (schools) => {
+  if (schools.length === 0) {
+    statusMessage = 'No results found'
+  }
+
+  return schools
+}
+
+const findSchools = (query, populateResults) => {
+  statusMessage = 'Loading...' // Shared state
+
+  window.fetch(`/api/schools?query=${query}&lead_school=true`)
```

Comment:
![image](https://user-images.githubusercontent.com/470137/116603383-c68d5500-a924-11eb-8331-d171d0beece7.png)


```suggestion
  window.fetch(`/api/schools?query=${encodeURIComponent(query)}&lead_school=true`)
```

Reply:
added